ext/mruby_engine/memory_pool.h

```diff
@@ -32,6 +32,8 @@ struct me_memory_pool *me_memory_pool_new(size_t capacity, struct me_memory_pool
 void me_memory_pool_destroy(struct me_memory_pool *self);
 
 size_t me_memory_pool_get_allocation(struct me_memory_pool *self);
+size_t me_memory_pool_get_arena(struct me_memory_pool *self);
+size_t me_memory_pool_get_hblkhd(struct me_memory_pool *self);
```

Comment:
What do you think of having one function that fills in a structure with _n_ fields instead of _n_ functions? I don't really mind though because this is temporary, right?